mcli/mcli-pretokenize-oci-upload.yaml

```diff
@@ -1,5 +1,5 @@
 name: c4-2k-pre-tokenized
-image: mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04
+image: mosaicml/llm-foundry:2.1.0_cu121_flash2-latest
```

Comment:
why are images 2.1?

Reply:
because the example yamls are pinned to the 0.5.0 release of llm foundry. we can update when we go to 0.6.0